src/chrome/content/rules/Electronic-Arts.xml

```diff
@@ -1,76 +1,58 @@
 <!--
 	Other Electronic Arts rulesets:
-
 		- BioWare.xml
 		- EA_Sports.com.xml
 		- Origin.com.xml
 		- Sim_City.com.xml
 
-	!functional:
-		- eu.dragonage.com		(cert: socialsignup.ea.com; shows that domain)
-		- (prod.)crysis.ea.com		(timeout)
-		- cdn.prod.crysis.ea.com	(Akamai; "An error occurred")
-		- eapressbox.ea.com		(timeout)
-		- forum.ea.com			(timeout)
-		- cdn.forum.ea.com		(Akamai; "An error occurred"; same data on forum)
-		- gpl.ea.com			(timeout)
-		- (www.)info.ea.com
-		- investor.ea.com		(cert: *.shareholder.com; 403)
-		- preferences.ea.com		(times out)
-		- eamobile.com			(cert: synergy-stage.eamobile.com; shows that domain's data)
-		- insideea.com			(ssl_error_rx_record_too_long)
+	Invalid certificate:
+		- eu.dragonage.com
+		- support.ea.com, equivalent to help.ea.com
+		- om.eamobile.com, equivalent to eamobile.122.2o7.net
+
+	Timed out:
+		- investor.ea.com
+		- resources.ea.com, equivalent to ssl.resources.ea.com
 -->
-<ruleset name="Electronic Arts (partial)" platform="mixedcontent">
 
+<ruleset name="Electronic Arts (partial)">
+	<!-- ea.com -->
 	<target host="ea.com"/>
-	<target host="*.ea.com"/>
+	<target host="www.ea.com" />
+	<target host="activate.ea.com" />
+	<target host="click.e.ea.com" />
+	<target host="fonts.ea.com" />
+	<target host="forum.ea.com" />
+	<target host="gpl.ea.com" />
+	<target host="help.ea.com" />
+	<target host="info.ea.com" />
+	<target host="jobs.ea.com" />
+	<target host="www.jobs.ea.com" />
+	<target host="profile.ea.com" />
+	<target host="resources.ea.com" />
+	<target host="ssl.resources.ea.com" />
+	<target host="support.ea.com" />
+	<target host="tos.ea.com" />
```

Comment:
Just an attempt to make this ruleset more complete:
```
answers.ea.com
accounts.ea.com
media.contentapi.ea.com
ir.ea.com
forums.ea.com
myaccount.ea.com
nds-network-nav.ea.com
news.ea.com
o.ea.com (m)
pl.ea.com
starwars.ea.com
forums.galaxy-of-heroes.starwars.ea.com
x.ea.com
```
Test urls for the CDNs can be found anywhere on the website.